JS-Algorithms/challenges.js

```diff
@@ -41,7 +49,16 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  let positiveIndex = Math.abs(index);
+
+  let result = [];
+  
+  for (let i = 0; i < COUNTRY_NAMES.length; i++) {
+    let country = COUNTRY_NAMES[(positiveIndex + i) % COUNTRY_NAMES.length];
```

Comment:
same question about const usage over let

Reply:
I overlooked the use of const over let, the  values that are not being reassigned should all be using const. Will keep that in mind for future challenges and projects.